framework/src/modules/chain/modules/blocks/process.js

```diff
@@ -537,68 +547,40 @@ Process.prototype.loadBlocksFromPeer = function(peer, cb) {
  * @returns {function} cb - Callback function from params (through setImmediate)
  * @returns {Object} cb.err - Error message if error occurred
  */
-Process.prototype.generateBlock = function(keypair, timestamp, cb) {
+Process.prototype.generateBlock = async function (keypair, timestamp, cb) {
 	// Get transactions that will be included in block
 	const transactions = modules.transactions.getUnconfirmedTransactionList(
 		false,
 		MAX_TRANSACTIONS_PER_BLOCK
 	);
-	const ready = [];
-
-	async.eachSeries(
-		transactions,
-		(transaction, eachSeriesCb) => {
-			modules.accounts.getAccount(
-				{ publicKey: transaction.senderPublicKey },
-				(err, sender) => {
-					if (err || !sender) {
-						return setImmediate(eachSeriesCb, 'Sender not found');
```

Comment:
Why is this check gone? Who is validating this now?

Reply:
`verifyTransactions` will call `transaction.apply` which calls `stateStore.getOrDefault` which creates a default account in case it does not exist. therefore this check is no longer required.